app/components/wallet/transactions/Transaction.js

```diff
@@ -184,7 +184,6 @@ type Props = {|
   +state: TxStatusCodesType,
   +selectedExplorer: SelectedExplorer,
   +assuranceLevel: AssuranceLevel,
-  +isLastInList: boolean,
```

Comment:
Why was this removed?

Reply:
reverted back - looks like it was a mistake